### DISCUSSION TITLE
Why do the windows with unmanage=off and grid applied do this on restart?

### DISCUSSION THREAD

**Discussion by szfkamil** [asmvik/yabai#2233] [Q&A] [answered]
As for 'this' from the title, see the video. They tile, until the `yabai -m rule --apply` from the bottom of the rc file gets executed.

Is there a way to make this not happen, or is this the way yabai just works? I noticed similar behaviour with other windows.

https://github.com/koekeishiya/yabai/assets/82778513/f1300d1e-1361-416f-b6d7-2f982e4e4b64

**Comment by asmvik** [comment, marked as answer]
Order matters. Put your rule commands above the config commands in your yabairc file.
